cmake_format/test/cmake-format.py

```diff
@@ -13,6 +13,9 @@
 # If true, separate function names from the parenthesis with a space
 separate_fn_name_with_space = False
 
+# list of directories which will be ignore during parsing
```

Comment:
@dkandyba `ignore -> ignored`

Reply:
@AByzhynar Fixed in: dc880b1cb71405b990ba4615189e39ccbd14f6a6